incubator/mattermost/values.yaml

```diff
@@ -0,0 +1,47 @@
+# Default values for mattermost.
+# This is a YAML-formatted file.
+# Declare variables to be passed into your templates.
+replicaCount: 1
+image:
+  repository: mattermost/mattermost-prod-app
+  tag: 3.10.0
+  pullPolicy: IfNotPresent
+service:
+  name: mattermost
+  type: ClusterIP
+  port: 80
+ingress:
+  enabled: false
+  # Used to create Ingress record (should used with service.type: ClusterIP).
+  hosts:
+    - chart-example.local
+  annotations:
+    # kubernetes.io/ingress.class: nginx
+    # kubernetes.io/tls-acme: "true"
+  tls:
+    # Secrets must be manually created in the namespace.
+    # - secretName: chart-example-tls
+    #   hosts:
+    #     - chart-example.local
+mattermost:
```

Comment:
Why is just the DB under `mattermost`? You could probably put everything under that so I'd just remove it.